hack/update_test_config.sh.bats

```diff
@@ -35,7 +35,7 @@ teardown() {
     [[ "${testConfigFile}" =~ "tests.build-image-version=latest" ]]
     [[ "${testConfigFile}" =~ "tests.services-image-version=latest" ]]
     [[ "${testConfigFile}" =~ "tests.runtime-application-image-version=latest" ]]
-    [[ "${testConfigFile}" =~ "tests.examples-ref=nightly-master" ]]
+    [[ "${testConfigFile}" =~ "tests.examples-ref=nightly-main" ]]
```

Comment:
please align it with @radtriste

Reply:
This can only be changed if kogito-examples has been migrated